SonarLint.VisualStudio.Integration.sln

```diff
@@ -34,13 +34,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		SonarLint.VsTargetVersion.props = SonarLint.VsTargetVersion.props
 	EndProjectSection
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".sonarlint", ".sonarlint", "{71E68181-C534-4C18-84AA-42A8C273414E}"
-	ProjectSection(SolutionItems) = preProject
-		src\SonarQube\SolutionBinding.sqconfig = src\SonarQube\SolutionBinding.sqconfig
-		src\SonarQube\sonarlint-visualstudioCSharp.ruleset = src\SonarQube\sonarlint-visualstudioCSharp.ruleset
-		src\SonarLint.xml = src\SonarLint.xml
-	EndProjectSection
-EndProject
```

Comment:
The migration wizard doesn't clean up the solution file so we have to do it manually (logged as #4432, marked as `Won't fix`.